sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1580,34 +1588,82 @@ def __init__(self, namespaces, port_mapping, main_thread_stop_event):
         self.port_mapping = copy.deepcopy(port_mapping)
         self.namespaces = namespaces
 
+    def get_dom_status_from_config_db(self, lport):
```

Comment:
@mihirpat1 suggest dom_status -> dom_polling?

Reply:
@prgeor I have addressed this now.